frontend/src/components/requisitionForm/RequisitionForm.tsx

```diff
@@ -126,7 +185,7 @@ const RequisitionForm: React.FunctionComponent = () => {
           <Col {...halfLayout}>
             <Form.Item
               name="vendor"
-              rules={[RULES.requiredRule]}
+              rules={[FORM_RULES.requiredRule]}
               label="Vendor"
             >
               <Select options={vendorOptions} showSearch optionFilterProp="label" loading={loading} />
```

Comment:
I can just make an issue so we can address this later, but the ability to create a vendor at the same time as opening a requisition would be nice